### DISCUSSION TITLE
Can't use overlay borders (gambattle) in Onion?

### DISCUSSION THREAD

**Discussion by kissdreams** [OnionUI/Onion#488] [General]
Just tried but simply displayed "Directory Not Found" in the ON-SCREEN OVERLAY opition, won't load any .cfg files though.
Is there any trick for using the overlay function i just don't know, or simlpy this function is not available in the Onion system?

**Comment by Sichroteph** [comment]
I believe it's for performance reasons

**Comment by kevmuller** [comment]
bump this plz, im trying to get my gba/gbc games looking real good and this is frustrating, im like trying to duplicate from the OG miyoo SD card to make this work
